src/classes/dropdown.py

```diff
@@ -13,13 +14,24 @@ def __init__(
         label: str = "",
         width: Optional[int] = None,
     ) -> None:
-        super().__init__()
+        super().__init__(
+            color=COLORS.MAIN_COLOR.value,  # צבע טקסט
+            bgcolor=COLORS.BACKGROUND_COLOR.value,
+            border_color=COLORS.MAIN_COLOR.value,
+            border_width=1,
+            border_radius=8,
+            filled=True
+        )
         self.value = value
         self.options = options
         self.label = label
         self.width = width
         self.on_change = on_change
         self.on_click = on_click
+        self.border_width = 2
```

Comment:
```suggestion
            border_width=2,
            border_radius=8,
            filled=True
        )
        self.value = value
        self.options = options
        self.label = label
        self.width = width
        self.on_change = on_change
        self.on_click = on_click
```

This border_width is not the same?